.gitignore

```diff
@@ -5,3 +5,8 @@
 # IDE-specific files
 .vscode/*
 .idea
+
+# This file is used in local development to pass an /app/.env
+# file to the container, for secrets. It should not be committed
+# to the repo because tests and CI/CD will not have an .env file.
+docker-compose.override.yml
```

Comment:
So, I wonder if the real issue here is where we keep the docker-compose file. Would there be any issue moving it to the app folder itself?

That's what we did on the simpler grants gov repo (this `api` folder is the equivalent of `app`): https://github.com/navapbc/simpler-grants-gov/tree/main/api

We do have a top-level docker-compose but just for the purposes of spinning up both the frontend and API together.

Reply:
i'm open to that

Reply:
I agree that inside the app folder seems more appropriate to me.